libexec/pyenv-which

```diff
@@ -2,27 +2,46 @@
 #
 # Summary: Display the full path to an executable
 #
-# Usage: pyenv which <command> [--nosystem]
+# Usage: pyenv which <command> [--nosystem] [--no-advice]
 #
 # Displays the full path to the executable that pyenv will invoke when
 # you run the given command.
 # Use --nosystem argument in case when you don't need to search command in the 
 # system environment.
-#
+# Use --skip-advice to skip outputing suggestions to stderr
```

Comment:
Can document the switch as internal to save us from having to maintain compatibility

Reply:
done